### ISSUE TITLE
commentsByPost graphql query not present

### ISSUE THREAD

**Issue by imshivam-gupta** [PalisadoesFoundation/talawa-api#1460]
**Describe the bug**
- Query called commentsByPost is not present in graphql queries which is expected from the talwa fetch comments service.

**To Reproduce**

-  https://youtu.be/86AzK_Y_GCQ?t=48

This issue is linked to [Issue 2190](https://github.com/PalisadoesFoundation/talawa/issues/2190)

**Expected behavior**
We need to have graphql query to fetch data as expected 

**Screenshots**
![image](https://github.com/PalisadoesFoundation/talawa/assets/93479496/9f121a29-5835-47dc-b555-385e4d6265b8)

**Comment by imshivam-gupta** [comment]
@noman2002 Can you please assign this issue to me

**Comment by imshivam-gupta** [comment]
I have checked the existing GraphQL schema and confirmed that it already satisfies the requirements for commentsByPost. There was no need for a separate query. I opened this issue initially to create APIs for the frontend, but I now realize that updating the frontend queries suffices. Thanks @xoldyckk  for providing this insight; it was overlooked by me earlier. I also noticed that the Talawa-API has many redundant queries, for which I will definitely create issues in the future. This way, we can maintain sustainable code without the need for additional queries. @palisadoes I am closing this pull request and will chnage the frontend query directly to fetch the comments.
